sdk/keyvault/azure-keyvault-certificates/tests/test_certificates_client.py

```diff
@@ -524,7 +535,6 @@ def test_get_pending_certificate_signing_request(self, vault_client, **kwargs):
     @VaultClientPreparer()
     def test_backup_restore(self, vault_client, **kwargs):
         policy = CertificatePolicy.get_default()
-        policy._san_upns = ["user_principal"]
```

Comment:
Why do we need to remove this? Shouldn't it just be  a rename?

Reply:
@schaabs this is just for the test, so I removed it. I can add something like "john.doe@domain.com" for the user principal and see if it works

Reply:
I'm going to switch the name of the private internal variable keeping track of the san user principal names to also be _san_user_principal_names to avoid confusion